platform-dom/src/main/java/com/softicar/platform/dom/elements/input/auto/DomAutoCompleteIdMapFactory.java

```diff
@@ -0,0 +1,51 @@
+package com.softicar.platform.dom.elements.input.auto;
+
+import com.softicar.platform.common.core.entity.EntityIdFromDisplayStringExtractor;
+import java.util.Map;
+import java.util.Optional;
+import java.util.TreeMap;
+
+/**
+ * Attempts to extract numeric identifiers (IDs) from the displayed strings of
+ * auto-complete elements.
```

Comment:
`elements` ... this got me confused: For a moment, I though this was about the DOM elements while in fact it's about the individual values that are available for selection.

Reply:
I see, but I called them `elements` in all other auto-complete classes...I can rename them to `values` in the next PR, what do you think?

Reply:
yup, I think that would be great

Reply:
i'm already on it